docs/docs/cmd/purview/retentionlabel/retentionlabel-add.md

```diff
@@ -0,0 +1,128 @@
+# purview retentionlabel add
+
+Create a retention label
+
+## Usage
+
+```sh
+m365 purview retentionlabel add [options]
+```
+
+## Options
+
+`-n, --displayName <displayName>`
+: The display name of the label.
+
+`--behaviorDuringRetentionPeriod <behaviorDuringRetentionPeriod>`
+: Specifies how the behavior of a document with this label should be during the retention period. Allowed values: `doNotRetain`, `retain`, `retainAsRecord`, `retainAsRegulatoryRecord`.
+
+`--actionAfterRetentionPeriod <actionAfterRetentionPeriod>`
+: Specifies the action to take on a document with this label applied after the retention period. Allowed values: `none`, `delete`, `startDispositionReview`.
+
+`--retentionDuration <retentionDuration>`
+: The number of days to retain the content.
+
+`-t, --retentionTrigger [retentionTrigger]`
+: Specifies whether the retention duration is calculated from the content creation date, labeled date, or last modification date. Allowed values: `dateLabeled`, `dateCreated`, `dateModified`, `dateOfEvent`. Defaults to `dateLabeled`.
+
+`--defaultRecordBehavior [defaultRecordBehavior]`
+: Specifies the locked or unlocked state of a record label when it is created. Allowed values: `startLocked`, `startUnlocked`, Defaults to `startLocked`.
+
+`--descriptionForUsers [descriptionForUsers]`
+: The label information for the user.
+
+`--descriptionForAdmins [descriptionForAdmins]`
+: The label information for the admin.
+
+`--labelToBeApplied [labelToBeApplied]`
+: Specifies the replacement label to be applied automatically after the retention period of the current label ends.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Create a retention label that retains documents and deletes them after one year.
+
+```sh
+m365 purview retentionlabel add --displayName 'some label' --behaviorDuringRetentionPeriod retain --actionAfterRetentionPeriod delete --retentionDuration 365
+```
+
+Create a retention label that retains documents as records and starts a disposition review one year after the last modification date.
+
+```sh
+m365 purview retentionlabel add --displayName 'some label' --behaviorDuringRetentionPeriod retainAsRecord --actionAfterRetentionPeriod startDispositionReview --retentionDuration 365 --retentionTrigger dateModified
+```
```

Comment:
@nanddeepn, @martinlingstuyl I saw your discussion on the related issue and I think no decision was made but I do agree with @nanddeepn  that we should either update this example or provide a different one. The current one also gives error in my case

![image](https://user-images.githubusercontent.com/58668583/211110387-e45201e0-2c6e-4813-8668-dfc7740a1365.png)

Reply:
Thank @Adam-it 
Any updates needed to the examples?

Reply:
Yes, if I understood Martin i guess we should update the examples to working once 😉. We may provide other examples or revisit this in future. 
Besides that all other is fine 👍

Reply:
Thanks @Adam-it
Updated the examples.